students/CCSt130/lesson03/mail_room_pt1.py

```diff
@@ -0,0 +1,500 @@
+# -*- coding: utf-8 -*-
+""" This code allows a user to update a list of donors, print
+all donations, average donations, and print a thank you email. 
+"""
+
+"""
+Lesson03 :: Mailroom Part 1
+@author: Chuck Stevens :: CCSt130
+Created on Sun Jun 16 19:32:15 2019
+"""
+
+import datetime # dates needed for comparison
+
+from collections import Counter
+
+from statistics import mean
+
+if __name__ == "__main__":
```

Comment:
this is usually at the bottom of the file and `main()` is defined outside of this (this simply executes the main())